pkg/configuration/memoized.go

```diff
@@ -0,0 +1,29 @@
+package configuration
```

Comment:
The filename seems to have typo `memoized.go` or `memorized.go`?

Reply:
I see that's probably fine, but `*_test.go` has definitely a typo then. :)

Reply:
This is fine. After: https://en.wikipedia.org/wiki/Memoization

Test file got typo.